.gitignore

```diff
@@ -24,10 +24,10 @@ _testmain.go
 *.test
 *.prof
 
-lnd
-lnd-debug
-lncli
-lncli-debug
+/lnd
```

Comment:
Why do we _ever_ want to include binaries?

Reply:
Problem is that this will match on folders as well. In the case of this PR, the files added to `cmd/lncli` wouldn't show up, as they were ignored.

Reply:
Hmm, ok never had that happen to me, but can't hurt I guess.